src/app/Http/Controllers/CrudFeatures/AjaxTable.php

```diff
@@ -23,6 +25,19 @@ public function search()
                     ->merge($this->crud->model->getKeyName())
                     ->toArray();
 
+        // dont pass an order by through the query, but send a default order if one doesn't exist in the request
+        $default_orders = $this->crud->query->getQuery()->orders;
+        $this->crud->query->getQuery()->orders = null;
+        $order = $request->input('order');
+        if (! $order) {
+            $order = [];
+            foreach ($default_orders as $default_order) {
+                $order[] = ['column' => (string) array_search($default_order['column'], $columns), 'dir' => $default_order['direction']];
+            }
+            // we have to add to $_POST as EloquentDatatables checks this not the request.
+            $_POST['order'] = $order;
```

Comment:
What if there already *is* an `order` in `$_POST`, though? Should we stomp it without saying anything? Keep it? Throw an exception?